common/charts/nginx-ingress/templates/clusterrole.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
   name: {{ template "nginx-ingress.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
```

Comment:
What is the plan for these changes to a thirdparty chart? Are we maintaining our own fork? Can we contribute these changes upstream and use the off-the-shelf version (...eventually -- I realize we're using a local copy "soft forked" from the upstream charts repo)?

(Maybe we discussed this before and I forgot?)

Reply:
We discussed this before – this is related to known issue that prevents resources to be deployed in a proper namespace, when `release_namespace` attribute is set for the Helm chart. The only workaround atm is to explicitly set namespace for all resources. In ideal world, I would prefer to switch to upstream version for all public charts.

Reply:
Right, I understand why the change is needed.

Can we contribute the change to upstream? Everyone who uses this chart has the same problem with helm's treatment of namespaces.

Having this change upstream enables us to switch to the official version later.

Reply:
Makes sense, let me check their contribution guide.